dspace-api/src/main/java/org/dspace/storage/bitstore/BitstreamStorageServiceImpl.java

```diff
@@ -303,6 +303,10 @@ public void cleanup(boolean deleteDbRecords, boolean verbose) throws SQLExceptio
                 commitCounter++;
                 if (commitCounter % 100 == 0) {
                     context.dispatchEvents();
+                    // Commit actual changes to DB after dispatch events
+                    System.out.print("Committing changes to the database...");
```

Comment:
@amgciadev : I'd recommend changing the text here slightly so that it's not *identical* to the "System.out.print" which occurs below on line 315 when `context.complete()` is called.  That way, you can tell which commit is running based on the output.

Maybe changing the text here to be "Performing incremental commit to the database..."?

Beyond that, the rest of this PR looks good.

Reply:
Done @tdonohue